### PR TITLE
Keep the three package.json version numbers in sync

### PR DESCRIPTION
**Pull request by dfellis**
This was noticed in the last release. A minor glitch which should be fixed by these two script changes

### CODE REVIEW

File: .develop.sh [hunk 1]
```diff
@@ -15,14 +15,27 @@ CONFIGVERSION="$(cat src/config/index.ts | sed "s/^  version:.*/version: '${VERS
...
 # Make sure the lockfiles are updated, too
 yarn
 
 # Make sure it's all formatted the way we want it
 yarn format
 
+# Same for the dashboard
+cd dashboard
+yarn
+yarn format
```
Comment (aguillenv) [inline .develop.sh:30]: nit: do we need this?

File: .version.sh [hunk 2]
```diff
@@ -17,15 +17,28 @@ CONFIGVERSION="$(cat src/config/index.ts | sed "s/^  version:.*/version: '${VERS
...
 # Make sure the lockfiles are updated, too
 yarn
 
 # Make sure it's all formatted the way we want it
 yarn format
 
+# Same for the dashboard
+cd dashboard
+yarn
+yarn format
```
Comment (aguillenv) [inline .version.sh:32]: same nit

**Review by aguillenv — APPROVED** [review]

**Review by depombo — APPROVED** [review]

File: .develop.sh (hunk 1 above)
Comment (dfellis) [reply .develop.sh:30]: Hmm... Does our formatter also format the `package.json` file? I know we've configured formatting to touch `.sql` files, for instance. I put that in there just to be safe that we don't accidentally auto-commit files that break our tests.

Comment (aguillenv) [reply .develop.sh:30]: the `yarn format` inside the `dashboard` dir calls `npx prettier --config .prettierrc 'src/**/*.{ts,tsx}' --write` so should not be a problem

Comment (dfellis) [reply .develop.sh:30]: Then I can remove it if you want. We'll just need to be careful to re-add it if we ever auto-format the `package.json` files.

Comment (dfellis) [inline .develop.sh:30]:
```suggestion
```

File: .version.sh (hunk 2 above)
Comment (dfellis) [inline .version.sh:32]:
```suggestion
```

**Pull request merged**